### PR TITLE
Add flag to toggle between a default day and night temperature

### PR DESCRIPTION
**Pull request by fac3plant**
Add `--toggle` flag, allowing users to toggle between night (4500 K) and day (6500 K) temperature, depending on current temperature.

### CODE REVIEW

File: src/xsct.c [hunk 1]
```diff
@@ -223,6 +225,20 @@ int main(int argc, char **argv)
...
+        temp = get_sct_for_screen(dpy, screen, crtc_specified, fdebug);
+        if (temp.temp > (TEMPERATURE_NORM - 100)) {
+          temp.temp = TEMPERATURE_NIGHT;
+          sct_for_screen(dpy, screen, crtc_specified, temp, fdebug);
+        }
+        else {
+          temp.temp = TEMPERATURE_NORM;
+          sct_for_screen(dpy, screen, crtc_specified, temp, fdebug);
+        }
+      }
```
Comment (faf0) [inline src/xsct.c]: Thanks!

Requires some style changes for consistency with the code style of this project:
- needs further indentation to match the level below
- should switch to `//` single-line comments as used elsewhere
- `toggle != 0` to match the style of comparisons elsewhere
- curlies are on its own line as elsewhere
- `sct_for_screen` call can be moved out to a call after `temp.temp` update; however, the temperature of all screens should be updated as done elsewhere

File: src/xsct.c [hunk 2]
```diff
@@ -223,6 +225,20 @@ int main(int argc, char **argv)
     }
     else
     {
+      /* Check if the temp is above 100 less than the norm and change to NIGHT if it is */
+      /* The threashold was chosen to give some room for varients in temp */
+      if (toggle == 1) {
+        screen = 0;
```
Comment (faf0) [inline src/xsct.c]: can we change do this for all screens, not only the first?

File: xsct.1 [hunk 3]
```diff
@@ -25,6 +25,9 @@ Shift temperature by temperature value
 .B -s, --screen N
 Zero-based index of screen to use.
 .TP
+.B -t, --toggle
+Toggle redshift on and off.
```
Comment (faf0) [inline xsct.1]:
```suggestion
Toggle between night and day temperature.
```

**Comment by faf0** [comment]
Let's also update `README.md` for the new flag.

File: src/xsct.c [hunk 4]
```diff
@@ -223,6 +225,25 @@ int main(int argc, char **argv)
     }
     else
     {
+      // Check if the temp is above 100 less than the norm and change to NIGHT if it is
+      // The threashold was chosen to give some room for varients in temp
```
Comment (faf0) [inline src/xsct.c]: looks like these need to be indented further to align with the code below

File: src/xsct.c [hunk 5]
```diff
@@ -223,6 +225,25 @@ int main(int argc, char **argv)
...
+            temp = get_sct_for_screen(dpy, screen, crtc_specified, fdebug);
+            if (temp.temp > (TEMPERATURE_NORM - 100))
+            {
+                temp.temp = TEMPERATURE_NIGHT;
+            }
+            else
+            {
+                temp.temp = TEMPERATURE_NORM;
+            }
+                sct_for_screen(dpy, screen, crtc_specified, temp, fdebug);
```
Comment (faf0) [inline src/xsct.c:243]: indented too far

**Review by faf0 — DISMISSED** [review]
Thanks for your contribution, @fac3plant ! Once indentation is consistent, we're good to merge.

**Comment by fac3plant** [comment]
Thank you, and my apologies for not paying attention to coding format better.

**Review by faf0 — APPROVED** [review]
Thanks again! Ready to merge.

**Pull request merged**